doc/source/myrocks/limitations.rst

```diff
@@ -53,3 +53,16 @@ You should also consider the following:
   For more information, see `Replication Formats
   <https://dev.mysql.com/doc/refman/5.7/en/replication-formats.html>`_.
 
+* When converting from large MyISAM/InnoDB tables, either by using the
+  ``ALTER`` or ``INSERT INTO SELECT`` statements it's recommended that you
+  check the :ref:`Data loading <myrocks_data_loading>` documentation and
+  create MyRocks tables as below (in case the table is sufficiently big it will
+  cause the server to consume all the memory and then be terminated by the OOM
+  killer):
+
```

Comment:
Again, maybe BOLD or highlight this somehow as this is the important point.

Reply:
I've added a same warning about that's in data_loading doc